src/Microsoft.VisualStudio.Threading.Analyzers.Tests/VSTHRD003UseJtfRunAsyncAnalyzerTests.cs

```diff
@@ -90,6 +90,90 @@ public void Test()
         }
 
         [Fact]
+        public void ReportWarningWhenTaskTIsReturnedDirectlyFromLambda()
+        {
+            var test = @"
```

Comment:
another way is to embed the code as a resource string. (Just drag the file to the resource editor), which might be slightly easier to edit/maintain